spec/moneyTrackerSpec.js

```diff
@@ -0,0 +1,73 @@
+describe("Moneytracker", function() {
+
+  var moneytracker;
+
+  beforeEach(function() {
+    moneytracker = new Moneytracker(null, 0, 0, 0);
+
+    spyOn(transactions, 'push');
+  });
+
+  describe("#balance", function() {
+    it("sets the balance to 0 on initialisation", function() {
+      expect(moneytracker.balance).toEqual(0);
+    });
+
+    it("gives the correct balance after a transaction has been made", function() {
+      moneytracker.deposit(100);
+      moneytracker.withdraw(25);
+      expect(moneytracker.balance).toEqual(75);
+    });
+  });
+
+  describe("#credit", function() {
+    it("sets the credit to 0 on initialisation", function() {
+      expect(moneytracker.credit).toEqual(0);
+    });
+  });
+
+  describe("#debit", function() {
+    it("sets the debit to 0 on initialisation", function() {
+      expect(moneytracker.debit).toEqual(0);
+    });
+  });
+
+  describe("#deposit", function() {
+    it("allows the user to deposit money", function() {
+      moneytracker.deposit(100);
+      expect(moneytracker.balance).toEqual(100);
+    });
+  });
+
+  describe("#withdraw", function() {
+    it("allows the user to withdraw money", function() {
```

Comment:
It feels like this code is talking in a domain language that isn't in the spec.  The spec doesn't talk about withdrawing, it talks about spendings.  It's generally good if the program reflects the domain language in the spec or stories.